test/recording_test.dart

```diff
@@ -448,46 +440,301 @@ void main() {
         });
       });
 
-      group('File', () {});
+      group('File', () {
+        test('create', () {});
```

Comment:
They're TODO - I have them as placeholders since I was enumerating all the methods I wanted to test...  I'll fill them in in a later change.